payload.adoc

```diff
@@ -255,14 +255,14 @@ optional modes supported by the encoder.
 |*subformat*| 2| 11 (support): Supporting information for the decoder
 |*ienable*| 1| Indicates if the instruction trace encoder is enabled
 |*encoder_mode*| N| Identifies trace algorithm Details and number of
-bits implementation dependent. Currently Branch trace is the only mode
+bits implementation dependent. Currently, Branch Trace is the only mode
 defined, indicated by the value 0.
 |*qual_status*| 2| 00 (no_change): No change to filter qualification +
-01 (ended_rep): Qualification ended, preceding *te_inst* sent explicitly to indicate last qualification
+01 (ended_rep): Qualification ended, preceding *te_inst* sent explicitly to indicate final qualification
```

Comment:
In reviewing where this text exists in the existing PDF, I notice that it doesn't properly show any of the 2-bit values before the parentheses.  That should be fixed so people know what the encodings are.

Reply:
That was a failure of the semi-automatic LaTeX to adoc conversion process that got missed at the time.  I had fixed many of these, but your comment prompted me to do a thorough review and I found a few more, which I have now fixed and included in the PR.